main.go

```diff
@@ -123,3 +129,43 @@ func main() {
 	}
 	log.Fatalln(gwServer.ListenAndServeTLS("", ""))
 }
+
+func runClient() {
+	addr := fmt.Sprintf("localhost:%d", *gRPCPort)
+	// Set up a connection to the server.
+	conn, err := grpc.Dial(addr,
+		grpc.WithTransportCredentials(credentials.NewClientTLSFromCert(insecure.CertPool, "")),
+	)
+	if err != nil {
+		log.Fatalf("did not connect: %v", err)
+	}
+	defer conn.Close()
+	c := pbExample.NewUserServiceClient(conn)
+
+	{
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+		defer cancel()
+
+		user := pbExample.User{ID: 42, Role: pbExample.Role_ADMIN}
+		_, err = c.AddUser(ctx, &user)
+		if err != nil {
+			log.Fatalf("could not create user: %v", err)
+		}
+		log.Infof("created user %+v", user)
+	}
```

Comment:
Why the explicit scope? I don't think I've ever seen this used in the wild before.